bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZprimeVBF_tau_tau/Zprime_tau_tau_M_1000p0_gl_0_gh_1_kv_1p0_run_card.dat

```diff
@@ -0,0 +1,251 @@
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  Zprime_tau_tau_M_1000p0_gl_0_gh_1_kv_1p0     = run_tag ! name of the run 
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  100000 = nevents ! Number of unweighted events requested 
+  84200   = iseed   ! rnd seed (0=assigned automatically=default))
```

Comment:
suggest 
5000 = nevents 
0 = iseed 
please use the same in all three scripts. ideally you would just upload template cards and a script that replaces the couplings. no need to upload 12 files. different files are just error-prone is settings might not be in sync. if you would use a template, all would be in sync.

Reply:
Hi @agrohsje , I am adding the templates. We did use the templates and as you said it would be good to look the templates only.